recipes-bsp/barebox/barebox.inc

```diff
@@ -88,10 +88,10 @@ do_deploy[vardepsexclude] = "DATETIME"
 do_deploy () {
 	install -d ${DEPLOYDIR}
 	if [ -e ${B}/barebox-flash-image ]; then
-		install -m 644 -t ${DEPLOYDIR}/ ${B}/barebox-flash-image
+		install -m 644 -t ${DEPLOYDIR}/barebox-flash-image${BAREBOX_IMAGE_SUFFIX} ${B}/barebox-flash-image
```

Comment:
This cannot work as `-t` argument must be a directory.

```
install -m 644 ${B}/barebox-flash-image ${DEPLOYDIR}/barebox-flash-image${BAREBOX_IMAGE_SUFFIX}
```

could work instead.

Reply:
Uhm yes. also `-T, --no-target-directory` seems useful here. Also rebased onto hardknott.